docs/classad-attributes/machine-classad-attributes.rst

```diff
@@ -60,83 +52,52 @@ Machine ClassAd Attributes
-
-``ClockMin``
+    Saturday. :classad-attribute:`ClockMin`
     The number of minutes passed since midnight.
 
-:index:`CondorLoadAvg<single: CondorLoadAvg; ClassAd machine attribute>`
-
-``CondorLoadAvg``
+:classad-attribute:`CondorLoadAvg`
     The load average contributed by HTCondor, either from remote jobs or
     running benchmarks.
 
-:index:`CondorVersion<single: CondorVersion; ClassAd machine attribute>`
-
-``CondorVersion``
+:classad-attribute:`CondorVersion`
     A string containing the HTCondor version number for the
     *condor_startd* daemon, the release date, and the build
     identification number.
 
-:index:`ConsoleIdle<single: ConsoleIdle; ClassAd machine attribute>`
-
-``ConsoleIdle``
+:classad-attribute:`ConsoleIdle`
     The number of seconds since activity on the system console keyboard
     or console mouse has last been detected. The value can be modified
     with ``SLOTS_CONNECTED_TO_CONSOLE`` :index:`SLOTS_CONNECTED_TO_CONSOLE` as defined in the
     :ref:`admin-manual/configuration-macros:condor_startd configuration
     file macros` section.
 
-:index:`Cpus<single: Cpus; ClassAd machine attribute>`
-
-``Cpus``
+:classad-attribute:`Cpus`
     The number of CPUs (cores) in this slot. It is 1 for a single CPU
     slot, 2 for a dual CPU slot, etc. For a partitionable slot, it is
     the remaining number of CPUs in the partitionable slot.
 
-:index:`CpuFamily<single: CpuFamily; ClassAd machine attribute>`
-
-``CpuFamily``
+:classad-attribute:`CpuFamily`
     On Linux machines, the Cpu family, as defined in the /proc/cpuinfo
-    file. :index:`CpuModel<single: CpuModel; ClassAd machine attribute>`
-
-``CpuModel``
+    file. :classad-attribute:`CpuModel`
     On Linux machines, the Cpu model number, as defined in the
     /proc/cpuinfo file.
 
-:index:`CpuCacheSize<single: CpuCacheSize; ClassAd machine attribute>`
-
-``CpuCacheSize``
+:classad-attribute:`CpuCacheSize`
     On Linux machines, the size of the L3 cache, in kbytes, as defined
     in the /proc/cpuinfo file.
 
-:index:`CurrentRank<single: CurrentRank; ClassAd machine attribute>`
-
-``CurrentRank``
+:classad-attribute:`CurrentRank`
     A float which represents this machine owner's affinity for running
     the HTCondor job which it is currently hosting. If not currently
     hosting an HTCondor job, ``CurrentRank`` is 0.0. When a machine is
     claimed, the attribute's value is computed by evaluating the
     machine's ``Rank`` expression with respect to the current job's
-    ClassAd. :index:`DetectedCpus<single: DetectedCpus; ClassAd machine attribute>`
-
-``DetectedCpus``
-    Set by the value of configuration variable ``DETECTED_CORES``
-
-:index:`DETECTED_CORES`
-:index:`DetectedMemory<single: DetectedMemory; ClassAd machine attribute>`
-
-``DetectedMemory``
-    Set by the value of configuration variable ``DETECTED_MEMORY``.
+    ClassAd. :classad-attribute:`DetectedCpus`.
     :index:`DETECTED_MEMORY` Specified in MiB.
```

Comment:
This is all messed up.  It needs a newline andthe whole body text for DetectedMemory andthe whole of DetectedCPUs at a minimum.  It should probably also fix the indexing of DETECTED_CORES and DETECTED_MEMORY.